pkg/cmd/dcos.go

```diff
@@ -3,14 +3,29 @@ package cmd
 
 import (
 	"github.com/dcos/dcos-cli/pkg/cli"
+	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 )
 
 // NewDCOSCommand creates the `dcos` command with its `auth`, `config`, and `cluster` subcommands.
 func NewDCOSCommand(ctx *cli.Context) *cobra.Command {
+	var verbose int
 	cmd := &cobra.Command{
 		Use: "dcos",
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			if verbose == 1 {
+				// -v sets the logger level to info.
+				ctx.Logger.SetLevel(logrus.InfoLevel)
+			} else if verbose > 1 {
+				// -vv sets the logger level to debug. This also happens for -vvv
+				// and above, in such cases we set the logging level to its maximum.
+				ctx.Logger.SetLevel(logrus.DebugLevel)
+			}
+		},
 	}
+
+	cmd.PersistentFlags().CountVarP(&verbose, "", "v", "verbose (-v, or -vv)")
```

Comment:
s/`verbose (-v, or -vv)`/`verbosity (-v or -vv)`. Also, the current output `-v, -- count   verbose (-v, or -vv)` is not very meaningful (what does `-- count` means?), is there any way we can customize it?